src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java

```diff
@@ -50,4 +51,55 @@ int test1(int badFormat1, int badFormat2, final int badFormat3) throws java.lang
 
   // Very long url with valid href: href    = "www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
   int validHrefWithWhiteSpaces = 54;
+
+  // @cs-: LineLength for +1 lines
+  int aaaarealllllllllllllllllyyyyyyyyyyylllllllloooooooooooooooonnnnnnnnnnnnnnnnnggggggggggvvvvvvaarriaaabllee = 99; // ok
+
+  // @cs-: LineLength for +1 lines
+  void longggggggggggggggggggggmethoooooooooooooooooooooodddddddddddddddddddddddddddddddddddddd(int x, int y) {} // ok
+
+  // @cs-: LineLength for +1 lines
+  void testingParametersNames(int areallllllyyyyyyyyyyyyyyyyyylonnnnggggggggggname, int anotherlongnameblahblahblah) {} // ok
+  // Above code is wrap-able. User should not use suppression for such cases
```

Comment:
make first parameter to be very loooong (not wrape-able)

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/0386838241ac8d778427e82fb9e215a991ad6c4e/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java#L110-L115

the first method isn't suppressed, second is suppressed